src/fragments/lib/push-notifications/ios/getting_started/30_create_app_delegate.mdx

```diff
@@ -0,0 +1,53 @@
+## Application Delegate
+
+APNs requires an application delegate to handle push notifications functionality via delegate methods. If your app does not already have an application delegate, you will need to create one:
+
+1. Create a class that subclasses `NSObject` and implements the `UIApplicationDelegate` (iOS) or `NSApplicationDelegate` (macOS) protocol.
+
+<BlockSwitcher>
+<Block name="iOS">
+
+```swift
+class AppDelegate: NSObject, UIApplicationDelegate {
+
+}
+```
+
+</Block>
+<Block name="macOS">
+
+```swift
+class AppDelegate: NSObject, NSApplicationDelegate {
+
+}
+```
+
+</Block>
+</BlockSwitcher>
+
+2. Add a property to your main app struct that identifies your application delegate.
```

Comment:
This is `SwiftUI` only, no?
Should we call that out somehow?

Reply:
I believe that's covered by the fact that this section only applies to people who don't already have an application delegate, which means they must be using SwiftUI.